test/jdk/java/awt/print/PrinterJob/PrintParenString.java

```diff
@@ -21,226 +21,86 @@
  * questions.
  */
 
-/**
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JOptionPane;
+
+/*
  * @test
  * @bug 4399442
  * @summary Brackets should be quoted in Postscript output
  * @key printer
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @run main/manual PrintParenString
  */
+public class PrintParenString implements Printable {
+    private static final String STR = "String containing unclosed parenthesis (.";
+    private static final String INSTRUCTIONS =
```

Comment:
```suggestion
    private static final String STR = "String containing unclosed parenthesis (.";

    private static final String INSTRUCTIONS =
```
I'd add a blank line between these two.